### PR TITLE
Exercises for lesson 3 labs

### PR DESCRIPTION
**Pull request by mmanc125uw**
exercises for lesson 3 labs.
3.1 slicing lab
3.2 list lab
3.3 string formatting lab

### CODE REVIEW

File: students/mmancini/lesson03/list_lab.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+def series1(in_lst):
+    result_lst = in_lst.copy()
+    print(result_lst)
+
+    fruit = input("Enter fruit name: ")
+    result_lst.append(fruit)
+    print('{0}'.format(result_lst))
+
+    num = input("Enter a number 1-5: ")
```
Comment (charlesrobison) [inline students/mmancini/lesson03/list_lab.py]: len() is a good idea to calculate the length of the list

File: students/mmancini/lesson03/list_lab.py [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+    print('{0}'.format(result_lst[ix]))
+
+    result_lst = ["BlueBerry"] + result_lst
+    result_lst.insert(0, "BlackBerry")
+    print('{0}'.format(result_lst))
+
+    for item in result_lst:
+        s = str(item)
+        s.startswith('P')
+        if item.startswith('P'):
```
Comment (charlesrobison) [inline students/mmancini/lesson03/list_lab.py]: upper() will catch case sensitive user input

File: students/mmancini/lesson03/list_lab.py [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+
+def series3(in_lst):
+    result_lst = in_lst.copy()
+    print(f"series 3, original list ", result_lst)
+
+    for item in in_lst:
+        ans = input("\nDo you like " + item.lower() + "? > ")
+        while ans != 'yes' and ans != 'no':
+            ans = input("\nPlease specify yes or no > ")
```
Comment (charlesrobison) [inline students/mmancini/lesson03/list_lab.py]: Nice

File: students/mmancini/lesson03/mailroom.py [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+    print("mailroom end")
+
+
+###################################
+
+
+# main, test funcs
+
+if __name__ == "__main__":
+    mailroom()
```
Comment (charlesrobison) [inline students/mmancini/lesson03/mailroom.py]: Works well, but missing a couple of things.  You want the user to be able to input an option, then continue to loop through until he/she quits.  Also, missing the sorting function - the report should have the donors sorted by total donated, descending.

**Comment by mmanc125uw** [comment]
Thank you Charles for your review and input.  Yes, I am still working on mailroom (much to do yet) and I will submit when I complete the assignment.

Thank You,
Mike

________________________________

**Pull request closed without merging**